testo-lwt.opam

```diff
@@ -1,6 +1,6 @@
 # This file is generated by dune, edit dune-project instead
 opam-version: "2.0"
-synopsis: "Extension of the Alcotest library for Lwt"
+synopsis: "Test framework for OCaml, Lwt variant"
 description: """
 Use this if the tests return Lwt promises and you can't make them synchronous
```

Comment:
In which case you can't wrap them in Lwt_main.run ?

Reply:
right